scrapy_webarchive/spidermiddlewares.py

```diff
@@ -83,6 +114,30 @@ def _is_disallowed_by_spider(self, url: str, spider: Spider) -> bool:
 
         return hasattr(spider, "archive_disallow_regexp") and not re.search(spider.archive_disallow_regexp, url)
 
+    @property
+    def _uri_template(self) -> Optional[str]:
+        """Generates the search pattern based on the export URI format."""
+
+        search_template = self.settings.get("SW_EXPORT_URI")
+
+        if not search_template:
+            return None
+
+        return search_template.replace('{spider}', self.spider_name)
```

Comment:
This is fine, but a bit of a shortcut: there could be other variables not dependant on time in the future, and then it would be easy to miss adding it here. At least some code comment would be good to explain why the spider name is replaced. I think ideally, one could ask to replace variables with or without the time variables, so the substitution functionality is at a single place.